_includes/html/desktop-table.html

```diff
@@ -32,7 +32,7 @@
                 {% assign image = website.domain | append: ".svg" | prepend: letter %}
               {% endif %}
 
-              <img class="logo" loading="lazy" src="/img/{{ image }}" alt="">
+              <img class="logo desktop-only" loading="lazy" src="/img/{{ image }}" alt="">
```

Comment:
Is this needed? We are in `desktop-table.html` after all.

Reply:
Yes apparently it is needed, otherwise some images are loaded even when the table is hidden. I don't quite understand why it happens or why this fixes it but it does 🤷‍♂️